crates/ironrdp-server/src/server.rs

```diff
@@ -160,15 +176,32 @@ impl RdpServer {
     where
         S: FramedWrite + FramedRead,
     {
-        let mut buffer = vec![0u8; 4096];
-        let mut encoder = UpdateEncoder::new();
+        debug!("Starting client loop");
+        self.io_channel_id = Some(result.io_channel_id);
```

Comment:
Issue (non-blocking): we never handle more than one client at a time, so there is no actual problem, but I feel mixed about this approach of updating a global state (as far as the server is concerned) for something which is only relevant for the current client. This could lead to confusion, especially since this state isn't reset after client disconnection, leaving a redundant "dummy" value.

Thought: that being said, we know it’s technically always the same value, so the "dummy" value will always be the "good" value.

Suggestion: two avenues:
- Remove the `io_channel_id` from `RdpServer`: it’s possible to just keep the value in the function of the body and to give it to `handle_x224`
- Go the "constant" route completely. I directed otherwise in a previous comment, but on second thought it was not bad all things considered. The only request I have is to keep things consistent, i.e., use the constant in the connector itself too (previously the approach was mixed).